src/index.html

```diff
@@ -7,12 +7,57 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Stars</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+
+    <div class="containerpai">
+    <div class="containerfilho">
+      <div class="linha1">
+        <div class="stars stars-0"></div>
+        <div class="stars stars-1"></div>
+        <div class="stars stars-2"></div>
+        <div class="stars stars-3"></div>
+        <div class="stars stars-4"></div>
+      </div>
+      <div class="linha2">
+        <div class="stars stars-0 preenchida"></div>
+        <div class="stars stars-1"></div>
+        <div class="stars stars-2"></div>
+        <div class="stars stars-3"></div>
+        <div class="stars stars-4"></div>
+      </div>
+      <div class="linha3">
+        <div class="stars stars-0 preenchida"></div>
```

Comment:
Besides wrong classes and mixed languages, your BEM naming isn't correct.
Please double-check the **Checklist**